service/controller/resource/nodes/upgrade.go

```diff
@@ -26,7 +27,9 @@ func AnyOutOfDate(ctx context.Context, destinationRelease string, releases []rel
 		return false, clientNotFoundError
 	}
 
-	nodeList, err := cc.Client.TenantCluster.K8s.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	nodeList, err := cc.Client.TenantCluster.K8s.CoreV1().Nodes().List(ctx, metav1.ListOptions{
```

Comment:
I'd be cool to use the controller client, but I don't want to delay the release, so I'll guess we can change it later.